flink-kubernetes-operator/src/test/java/org/apache/flink/kubernetes/operator/service/StandaloneFlinkServiceTest.java

```diff
@@ -184,6 +184,125 @@ public void testReactiveScale() throws Exception {
                         buildConfig(flinkDeployment, configuration)));
     }
 
+    @Test
+    public void testTMReplicaScaleApplication() throws Exception {
```

Comment:
I think the new tests are good enough and you can simply remove the old unit test. That is covered in the one you added